bitrise.yml

```diff
@@ -1452,5 +1490,7 @@ trigger_map:
   pipeline: pipeline_multiple_shards
 - pull_request_target_branch: epic-branch/*
   pipeline: pipeline_multiple_shards
+- push_branch: jjSDET/mte-755
+    workflow: Jackie-perf-testing
```

Comment:
Nit: indentation

Reply:
Actually you don't have to add this but to modify L1490 with your workflow name:
```
pull_request_target_branch: main
workflow: your_workflow_name
```